new-lamassu-admin/src/pages/UserManagement/modals/ChangeRoleModal.js

```diff
@@ -0,0 +1,67 @@
+import { makeStyles } from '@material-ui/core/styles'
+import React from 'react'
+
+import Modal from 'src/components/Modal'
+import { Button } from 'src/components/buttons'
+import { Info2, P } from 'src/components/typography'
+
+import styles from '../UserManagement.styles'
+
+const useStyles = makeStyles(styles)
+
+const ChangeRoleModal = ({
+  showModal,
+  toggleModal,
+  user,
+  confirm,
+  inputConfirmToggle,
+  setAction
+}) => {
+  const classes = useStyles()
+
+  const handleClose = () => {
+    toggleModal()
+  }
+
+  return (
+    <>
+      {showModal && (
```

Comment:
(this comment applies to every other modal you have)

anyway this is a nitpick that I don't even know if it's worth it to point out, but basically whenever you have this type of statement you can also do

```
if (!showModal) return null
return <Modal>......</Modal>
```

or even 

`return (showModal && (<Modal>......</Modal>))`

But this is more of a readability choice than anything else, do as you (or Rafael) thinks it's best

Reply:
I can make this change on all modals except on the create user modal, as it kind of handles two modals and it is a bit more complicated than this change